prime-router/docs/getting-started/running-tests.md

```diff
@@ -0,0 +1,96 @@
+# Running ReportStream Tests
+
+## Smoke tests
+
+You should be able to run tests now to confirm that everything is working.
+
+```bash
+# Smoke test checks that most of the system is working
+./gradlew testSmoke
+```
+
+## Running the unit tests
+
+The build will run the unit tests for you when you invoke `./gradlew package`. However, you may sometimes want to invoke them explicitly. Use this command to do run the Unit Tests manually:
+
+```bash
+cd ./prime-router
+./gradlew test
+# Or to force the tests to run
+./gradlew test -Pforcetest
+```
+
+## Data conversion quick test
+
+The quick test is meant to test the data conversion and generation code. Use this following command to run all quick tests, which you should do as part of a Pull Request:
+
+```bash
+./quick-test.sh all
+```
+
+## Running the end-to-end tests locally
+
+End-to-end tests check if the deployed system is configured correctly. The tests use an organization called IGNORE for running the tests. In order to successfully run the end-to-end tests, you will need to:
+
+1. Have built successfully
```

Comment:
These steps should only be required the first time I think?

Reply:
Thanks for pointing this out. I didn't read through this doc close enough while moving it. I've made a number of updates, mostly removing all the references to things which are done on setup: https://github.com/CDCgov/prime-reportstream/commit/85ccca8595ee4a0171b3dbb5262e91a46c273111#diff-a7323cfd168364ba6202e5e97106bfc980647052121ae1f3733cd422215a585f